test/modules/TestResultCoordinator/Reports/DirectMethod/DirectMethodReport.cs

```diff
@@ -11,8 +11,8 @@ class DirectMethodReport : TestResultReportBase
     {
         public DirectMethodReport(
             string trackingId,
-            string expectedSource,
-            string actualSource,
+            string senderSource,
+            Option<string> receiverSource,
```

Comment:
If serialize this report, the json for receiverSource is an object containing Value and HasValue properties.  Or should we use string and allows empty string instead.  You decide.

Reply:
I like option because it explicitly lets us know that ReceiverSource is optional.